arm/Microsoft.Network/virtualNetworks/.parameters/vnetPeering.parameters.json

```diff
@@ -123,17 +25,11 @@
                     "allowForwardedTraffic": true,
                     "allowGatewayTransit": false,
                     "allowVirtualNetworkAccess": true,
-                    "useRemoteGateways": false
-                }
-            ]
-        },
-        "roleAssignments": {
-            "value": [
-                {
-                    "roleDefinitionIdOrName": "Reader",
-                    "principalIds": [
-                        "<<deploymentSpId>>"
-                    ]
+                    "useRemoteGateways": false,
```

Comment:
This is a bit confusing using both `remote` and `reverse`. I feel this is evolving more into a construct rather than a module as there is custom logic. Happy to be course corrected

Reply:
See other comment